app/src/main/java/com/vero/tasky/splash_feature/presentation/SplashScreen.kt

```diff
@@ -0,0 +1,45 @@
+package com.vero.tasky.splash_feature.presentation
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.Icon
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import com.vero.tasky.R
+import com.vero.tasky.core.presentation.navigation.Screens
+import kotlinx.coroutines.delay
+
+@Composable
+fun SplashScreen(
```

Comment:
No need for this

You can use Android's new SplashScreen library, that already does everything

Philipp has a video about it